File: exercicios/R/1020/1020.r
input = file('stdin','r')
ano = 0
mes =0
dia =0
valor = as.integer(readLines(input, n=1))

if(valor >= 365){
    ano = as.integer(valor / 365)
    valor = valor - (ano * 365)
}
if(valor >= 30 & valor < 365){
    mes = as.integer(valor / 30)
    valor = valor - (mes * 30)
}
if( valor < 60){
    dia = valor
}
write(sprintf("%d ano(s)",ano),'')
write(sprintf("%d mes(es)",mes),'')
write(sprintf("%d dia(s)",dia),'')

File: exercicios/JavaScript/1020/1020.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

const itens = [365, 30, 1]
var qt = 0;
var valor = parseInt(lines.shift());

itens.forEach(itens => {
    qt = Math.abs(valor / itens);

    qt = qt - qt%1;

    valor = valor - (qt * itens);

    if(itens == 365){
        console.log(qt + " ano(s)")
    }else if(itens == 30){
        console.log(qt + " mes(es)")
    }else if(itens == 1){
        console.log(qt + " dia(s)")
    }
});
